spi/transfer-spi/src/main/java/org/eclipse/dataspaceconnector/spi/types/domain/transfer/TransferProcess.java

```diff
@@ -100,6 +100,7 @@ public class TransferProcess implements TraceCarrier {
 
     private String id;
     private Type type = Type.CONSUMER;
+    private long createdTimestamp;
```

Comment:
nit: align member declaration of `createdTimestamp` to declaration of stateTimestamp

Reply:
Doesn't it make more sense as now to have all `state*` fields together?

```java
    private long createdTimestamp;
    private int state;
    private int stateCount = UNSAVED.code();
    private long stateTimestamp;
```